client/state/reader/lists/selectors.js

```diff
@@ -78,10 +78,7 @@ export const getSubscribedLists = createSelector(
  * @returns {boolean}        Whether lists are being requested
  */
 export function isUpdatedList( state, listId ) {
-	if ( ! has( state, 'reader.lists.updatedLists' ) ) {
-		return false;
-	}
-	return includes( state.reader.lists.updatedLists, listId );
+	return state.reader.lists.updatedLists.includes( listId );
```

Comment:
`updatedLists` and the entire tree up top is guaranteed to exist. Same for all instances below.